Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.GPU.sb.in

```diff
@@ -772,6 +772,9 @@
         SYS_fcntl
         SYS_fcntl_nocancel
         SYS_fileport_makefd
+#if PLATFORM(VISION) && ENABLE(MODEL_PROCESS)
+        SYS_fileport_makeport
```

Comment:
Change looks good to me, just want to double check with @pvollan , @smfr , or @hortont424 on the sandbox change